### ISSUE TITLE
Add parameter @case_sensitive to glob and rglob in pathlib

### ISSUE THREAD

**Issue by ca34f2d8-6192-4121-8c1a-db40dbb4f438** [python/cpython#81079]
BPO | [36898](https://bugs.python.org/issue36898)
--- | :---
Nosy | @serhiy-storchaka
PRs | <li>python/cpython#13274</li>
Files | <li>[pathlib.py](https://bugs.python.org/file48328/pathlib.py "Uploaded as text/plain at 2019-05-13.07:17:53 by Chuang Men")</li>

<sup>*Note: these values reflect the state of the issue at the time it was migrated and might not reflect the current state.*</sup>

<details><summary>Show more details</summary><p>

GitHub fields:
```python
assignee = None
closed_at = None
created_at = <Date 2019-05-13.07:17:56.195>
labels = ['3.8', 'type-feature', 'library']
title = 'Add parameter @case_sensitive to glob and rglob in pathlib'
updated_at = <Date 2019-05-13.07:58:11.844>
user = 'https://bugs.python.org/ChuangMen'
```

bugs.python.org fields:
```python
activity = <Date 2019-05-13.07:58:11.844>
actor = 'Chuang Men'
assignee = 'none'
closed = False
closed_date = None
closer = None
components = ['Library (Lib)']
creation = <Date 2019-05-13.07:17:56.195>
creator = 'Chuang Men'
dependencies = []
files = ['48328']
hgrepos = []
issue_num = 36898
keywords = []
message_count = 3.0
messages = ['342284', '342287', '342289']
nosy_count = 2.0
nosy_names = ['serhiy.storchaka', 'Chuang Men']
pr_nums = ['13274']
priority = 'normal'
resolution = None
stage = 'patch review'
status = 'open'
superseder = None
type = 'enhancement'
url = 'https://bugs.python.org/issue36898'
versions = ['Python 3.8']
```

</p></details>

### Linked PRs
* gh-102710

**Comment by barneygale** [comment]
This should be straightforward to add once #102616 lands.

**Comment by barneygale** [comment]
PR available: #102710

**Comment by barneygale** [comment]
Implemented in 3.12: #102710 / https://github.com/python/cpython/commit/8100be5535073a5442c2b8c68dcb2093ee69433d
